package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -114,3 +117,17 @@ def load_data(
 
     context = load_context(project_path=project_path, env=env)
     return context.catalog, context.pipelines
+
+def load_all_versions(filepath: str) -> Dict[str, any]:
+    version_list = [x for x in Path(filepath).iterdir() if x.is_dir()]
```

Comment:
I would recommend using clearer variable names instead of `x`, `i` etc.. In core Kedro we actually have a lint check for that. But in general it helps to understand the code more easily if proper names are given. So in this case I'd probably do `path for path in Path(filepath).iterdir() ...`